doc/source/merging.rst

```diff
@@ -78,34 +78,40 @@ some configurable handling of "what to do with the other axes":
 ::
 
     pd.concat(objs, axis=0, join='outer', join_axes=None, ignore_index=False,
-              keys=None, levels=None, names=None, verify_integrity=False)
+              keys=None, levels=None, names=None, verify_integrity=False,
+              union_categoricals=False, copy=True)
```

Comment:
thx. you're right that suggested name is more descriptive. my intension is let users know it works the same as `union_categoricals` (which is public), not another "less strict" rule.

leave it a while for any feedbacks:)